dev/docker_data/run.py

```diff
@@ -553,6 +558,18 @@ def load_cpac_data_config(data_config_file, participant_label,
             c['pipeline_setup']['system_config'][
                 'num_participants_at_once'] = 1
 
+    ##### Num_ants_threads code added here #####
```

Comment:
This code block probably doesn't need to be framed with this comment header and https://github.com/FCP-INDI/C-PAC/blob/8dc756623d9f7853dd02c587c9eae24971aa6217/dev/docker_data/run.py#L571 footer, although it does make it easier to read, so I'm not opposed to leaving it in

Reply:
removed these `#`